.github/workflows/python-app.yml

```diff
@@ -46,7 +46,9 @@ jobs:
 
     - name: Test with pytest
       run: |
-        DISPATCHER_MOCK_KB=yes python -m coverage run --source=cdci_osa_plugin -m pytest tests -sv --full-trace --log-cli-level=DEBUG 
+        DISPATCHER_MOCK_KB=yes  python -m coverage run --source=cdci_osa_plugin -m pytest tests -sv --full-trace --log-cli-level=DEBUG -m "not plugin_test_conf"
+    - run: |
```

Comment:
this one has no name now 
![image](https://user-images.githubusercontent.com/3909535/123780207-3f803d00-d8d3-11eb-93a5-ee7622ad19cf.png)

Reply:
Ah ok, I thought it would take the name above